mediachain/transactor/client.py

```diff
@@ -78,8 +78,9 @@ def journal_stream(self, catchup=True, timeout=None, event_map_fn=None):
         :return:
         """
         req = Transactor_pb2.JournalStreamRequest()
-        stream = self.client.JournalStream(req, timeout)
-        follower = BlockchainFollower(stream, catchup,
+        def open_stream():
```

Comment:
seems like a prime candidate for writing as a closure like `lambda: self.client.JournalStream(req, timeout)`